app/services/distribution_itemized_breakdown_service.rb

```diff
@@ -64,16 +61,12 @@ def distributions
   def current_onhand_quantities(inventory)
     if inventory
```

Comment:
@dorner I think this condition is no longer needed -- OK to blow up if inventory is nil now

Reply:
updated